### PR TITLE
Parse evaluation data from .metrics artifacts in taskcluster

### LINKED ISSUE #518: Issues with uploaded experiments
Some weird things I noticed in https://wandb.ai/moz-translations/lt-en:

- teacher-ensemble evals is empty
- group logs doesn't have any metrics
- group logs is missing for some groups
- quantized is missing
This group is listed in the docs https://firefox-ci-tc.services.mozilla.com/tasks/groups/KYZvXC6jTPG7A4j80SrYlQ and have a lot of evals that we don't see in W&B

### PR DESCRIPTION
**Pull request by vrigal**
Closes #518

Example with Taskcluster group `KYZvXC6jTPG7A4j80SrYlQ`:
https://wandb.ai/teklia/565-lt-en

### CODE REVIEW

File: tests/test_tracking_cli.py [hunk 1]
```diff
@@ -48,6 +48,7 @@ def samples_dir():
             "unittest",
         ],
         taskcluster_secret=None,
+        wandb_publication=True,
```
Comment (eu9ene) [inline tests/test_tracking_cli.py:51]: is it supposed to stay on?

**Review by eu9ene — APPROVED** [review]

**Comment by eu9ene** [comment]
> Closes #518
> 
> Example with Taskcluster group `KYZvXC6jTPG7A4j80SrYlQ`: https://wandb.ai/teklia/565-lt-en

I don't know if it's related to this fix, but I noticed that the teacher is called `teacher` and not `teacher-1` which will make it hard to compare with other runs.

File: tests/test_tracking_cli.py (hunk 1 above)
Comment (vrigal) [reply tests/test_tracking_cli.py:51]: No, it has been merge in #567, I'm rebasing.

**Comment by eu9ene** [comment]
> The teacher-1 run is only mentioned as `teacher` on Taskcluster. @eu9ene could you fill an issue with the required renaming(s) please ?

ok, I filed https://github.com/mozilla/firefox-translations-training/issues/573

**Pull request merged**